dist/spec.yaml

```diff
@@ -8772,6 +9258,7 @@ definitions:
         BOOKMARK: '#/definitions/BookmarkApplication'
         BROWSER_PLUGIN: '#/definitions/BrowserPluginApplication'
         OPENID_CONNECT: '#/definitions/OpenIdConnectApplication'
+        ORG2ORG: '#/definitions/Org2OrgApplication'
```

Comment:
We should remove this mapping and make sure Org2OrgApplication is still generated.